src/hb_layout.rs

```diff
@@ -150,6 +150,13 @@ pub fn layout_fragment(
     let hb_face = HbFace::new(font);
     unsafe {
         let hb_font = hb_font_create(hb_face.hb_face);
+        if !font.location.is_empty() {
+            hb_font_set_variations(
+                hb_font,
+                get_variation_data(font).as_ptr(),
```

Comment:
Even though this is technically safe according to my experiments on the rust playground, I get very nervous passing a pointer to a temporary value as an argument here. Can we add a `let variations = get_variation_data(font);` before calling `hb_font_set_variations` to make the lifetime more clear?